src/hooks/tradingView/useTradingView.ts

```diff
@@ -88,7 +88,7 @@ export const useTradingView = ({
     [marketId: string]: number | undefined;
   }>({});
   const { tickSizeDecimals: tickSizeDecimalsAbacus } = orEmptyObj(
-    useAppSelector(getCurrentMarketConfig)
+    useAppSelector(BonsaiHelpers.currentMarket.stableMarketInfo)
```

Comment:
This is a weird hook. It is expecting indexer tickSizeDecimals as well as Abacus ones. I understand replacing the `tickSizeDecimalsAbacus` but maybe we can deprecate needing a fallback here